### PR TITLE
Add netzpolitik.org publisher

### PR DESCRIPTION
**Pull request by SebChmie**

### CODE REVIEW

File: src/fundus/publishers/de/netzpolitik_org.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    generic_date_parsing,
+    generic_topic_parsing,
+    parse_title_from_root,
+    strip_nodes_to_text,
+)
+
+
+class NetzpolitikOrgParser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector = CSSSelector("div.entry-content > p")
```
Comment (MaxDall) [inline src/fundus/publishers/de/netzpolitik_org.py:18]: Judging by [this article](https://netzpolitik.org/2024/mastodon-und-peertube-eu-macht-fediverse-dicht-weil-niemand-die-server-betreiben-will/) the selector seems to have problems extracting text wrapped in `blockquote` tags.

File: src/fundus/publishers/de/__init__.py [hunk 2]
```diff
@@ -30,6 +31,13 @@
 
 # noinspection PyPep8Naming
 class DE(PublisherEnum):
+    NetzpolitikOrg = PublisherSpec(
+        name="netzpolitik.org",
+        domain="https://netzpolitik.org/",
+        sources=[Sitemap("https://netzpolitik.org/sitemap.xml"), RSSFeed("https://netzpolitik.org/feed/")],
```
Comment (MaxDall) [inline src/fundus/publishers/de/__init__.py:37]: Could you add a filter to the sitemap by utilizing the `sitemap_filter` parameter of `Sitemap` to request only pages of type `sitemap_posttype_post`. You can do this by utilizing `regex_filter` and `inverse` from the `fundus.scraping.filter` module.

**Review by MaxDall — CHANGES_REQUESTED** [review]
Thanks for adding 👍 This one looks really good and thank you that you made the effort to understand how Fundus works. I can tell from your PR :)

File: src/fundus/publishers/de/__init__.py (hunk 2 above)
Comment (SebChmie) [reply src/fundus/publishers/de/__init__.py]: Did you mean `sitemap-posttype-post` instead of `sitemap_posttype_post`?

**Review by MaxDall — APPROVED** [review]

**Review by addie9800 — APPROVED** [review]

**Pull request merged**